app/assets/javascripts/active_admin/application.js.coffee

```diff
@@ -1,10 +1,19 @@
 # Initializers
 $(document).on 'ready page:load', ->
   # jQuery datepickers (also evaluates dynamically added HTML)
-  $(document).on 'focus', '.datepicker:not(.hasDatepicker)', ->
+  datepicker_selector = 'input.datepicker:not(.hasDatepicker)'
+
+  $(document).on 'focus', datepicker_selector, ->
+    $input = $(@)
+
+    # Only applying datepicker to compatible browsers
+    is_web_kit = /WebKit/.test(navigator.userAgent)
+    is_date_type = $input.attr('type') is 'date'
+    return if is_web_kit and is_date_type
```

Comment:
@timoschilling Not sure I follow here. Do you mean line 12 should be changed to:

`return if is_web_kit and $input.type isnt "text"`?